tests/Doctrine/Tests/Common/Proxy/ProxyGeneratorTest.php

```diff
@@ -525,6 +525,28 @@ public function testPhp81NeverType()
         );
     }
 
+    /**
+     * @requires PHP >= 8.1.0
+     */
+    public function testEnumDefaultInPublicProperty() : void
+    {
+        $className = Php81EnumPublicPropertyType::class;
+
+        if ( ! class_exists('Doctrine\Tests\Common\ProxyProxy\__CG__\Php81EnumPublicPropertyType', false)) {
+            $metadata = $this->createClassMetadata($className, ['id']);
+
+            $metadata->expects($this->any())->method('hasField')->will($this->returnValue(true));
```

Comment:
`->expects($this->any()` is a no-op

```suggestion
            $metadata->method('hasField')->will($this->returnValue(true));
```